doc/rados/operations/stretch-mode.rst

```diff
@@ -8,110 +8,114 @@ Stretch Clusters
 Stretch Clusters
 ================
 
-A stretch cluster is a cluster that has servers in geographically separated
-data centers, distributed over a WAN. Stretch clusters have LAN-like high-speed
-and low-latency connections, but limited links. Stretch clusters have a higher
+A stretch cluster spreads hosts across geographically separated
+data centers, with specific internal strategies to robustly localize
+network traffic and respond to failures.
+Stretch clusters have a higher
 likelihood of (possibly asymmetric) network splits, and a higher likelihood of
 temporary or complete loss of an entire data center (which can represent
 one-third to one-half of the total cluster).
 
 Ceph is designed with the expectation that all parts of its network and cluster
 will be reliable and that failures will be distributed randomly across the
-CRUSH map. Even if a switch goes down and causes the loss of many OSDs, Ceph is
+CRUSH topology. If a host or network switch goes down and causes the loss of many OSDs, Ceph is
 designed so that the remaining OSDs and monitors will route around such a loss. 
 
 Sometimes this cannot be relied upon. If you have a "stretched-cluster"
 deployment in which much of your cluster is behind a single network component,
-you might need to use **stretch mode** to ensure data integrity.
+you might need to use the explicit *stretch mode* to ensure data integrity.
 
 We will here consider two standard configurations: a configuration with two
 data centers (or, in clouds, two availability zones), and a configuration with
-three data centers (or, in clouds, three availability zones).
+three data centers.
 
-In the two-site configuration, Ceph expects each of the sites to hold a copy of
-the data, and Ceph also expects there to be a third site that has a tiebreaker
-monitor. This tiebreaker monitor picks a winner if the network connection fails
-and both data centers remain alive.
+In the two-site configuration, Ceph arranges for each site to hold a copy of
+the data, with a third site that has a tiebreaker (arbiter, witness)
```

Comment:
I am always chary of "with-" clauses, because they run the risk of looking instrumental when in fact the word "with" is intended by the writer as a coordinating conjunction. But again, I will not permit this to hold up the merging and backporting of this PR.